testing/wine-mono/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Maintainer: Jens Staal <staal1978@gmail.com>
+pkgname=wine-mono
+pkgver=4.6.3
+pkgrel=0
+pkgdesc="Wine's built-in replacement for Microsoft's .NET Framew"
+url="http://wiki.winehq.org/Mono"
+arch="any"
+license="GPL"
+depends="wine"
+makedepends=""
+install=""
+subpackages=""
+source="https://dl.winehq.org/wine/wine-mono/${pkgver}/wine-mono-${pkgver}.msi"
+
+_builddir=
+prepare() {
+	mkdir -p "$srcdir"
```

Comment:
no point at all. I copied the APKBUILD from wine_gecko and this must have stayed there... I will remove it.